Public/Src/Cache/ContentStore/Distributed/NuCache/ContentLocationDatabase.cs

```diff
@@ -658,10 +658,17 @@ private void LogEntryDeletion(OperationContext context, ShortHash hash, ContentL
         /// <param name="contentHashListWithDeterminism">
         ///     The value, and associated determinism guarantee, to store.
         /// </param>
+        /// <param name="contentSession">
+        ///     The content session over which this operation is happening
+        /// </param>
+        /// <param name="cts">
+        ///     A token that can signal this call should return as soon as possible.
+        /// </param>
         /// <returns>
         ///     Result providing the call's completion status.
         /// </returns>
-        public abstract AddOrGetContentHashListResult AddOrGetContentHashList(OperationContext context, StrongFingerprint strongFingerprint, ContentHashListWithDeterminism contentHashListWithDeterminism);
+        public abstract Task<AddOrGetContentHashListResult> AddOrGetContentHashListAsync(OperationContext context, StrongFingerprint strongFingerprint, ContentHashListWithDeterminism contentHashListWithDeterminism, IContentSession contentSession,

```

Comment:
Would you mind placing each argument on a line of it's own, or keep them all on one line?